package.json

```diff
@@ -33,11 +33,13 @@
     "circular-json": "^0.1.6",
     "crypto": "^0.0.3",
     "es6-promise": "^2.0.0",
+    "forge-min": "^0.6.20",
```

Comment:
Sure! We're only using it for keygen, so pretty easy to replace. It was copied from Colony so I knew it worked. Lemme file an issue to replace.
https://github.com/uProxy/uproxy/issues/2051